src/client/src/rt-platforms/browser/browser.ts

```diff
@@ -17,14 +20,22 @@ const isPWA = () =>
   (window.matchMedia && window.matchMedia('(display-mode: standalone)').matches) ||
   (window.navigator as Navigator).standalone
 
-export default class Browser extends BasePlatformAdapter {
+export default class Browser implements Platform {
   readonly name = 'browser'
   readonly type = 'browser'
-
   readonly allowTearOff = !isRunningInIE() && !isPWA()
-  window = {
```

Comment:
notice here that instead of inline `close` method, we are now wrapping web window into default `PlartfromWindow` with the help of `createDefaultPlatformWindow`. This is done so that later we can use enhanced platfrom window APIs (bring to front, etc)